src/ray/raylet/node_manager.cc

```diff
@@ -541,8 +543,10 @@ void NodeManager::ProcessClientMessage(
   if (registered_worker && registered_worker->IsDead()) {
     // For a worker that is marked as dead (because the driver has died already),
     // all the messages are ignored except DisconnectClient.
-    if (static_cast<protocol::MessageType>(message_type) !=
-        protocol::MessageType::DisconnectClient) {
+    if ((static_cast<protocol::MessageType>(message_type) !=
```

Comment:
minor, but `static_cast<protocol::MessageType>(message_type)` comes up in 3 places in this function I think. Maybe we can do it once earlier on?